docs/parcels.md

```diff
@@ -0,0 +1,109 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be controlled imperatively 
+by an application, Parcels use similar methodology as Applications, and can be considered an advanced single spa feature so we recommend [starting with Applications.](/docs/applications.md)
+A parcel can be as large as an application or as small as an component and written in 
+any language as long as it exports the correct lifecycle events. In a single-spa world, your SPA contains 
+many registered applications and potentially many parcels. Usually you'll want to mount a parcel within the 
+context of an application, this will automatically unmount the parcel when the application is unmounted.
+
+## Parcel Lifecycles
+To read more about lifecycles and how single-spa handles the concept start with [applications](/docs/applications.md#registered-application-lifecycle)
+
+### Bootstrap
+This lifecycle function will be called once, right before the parcel is
```

Comment:
If i remember correctly, this is a "Must" right? If so, maybe reword: "Before the parcel is mounted, the Bootstrap lifecycle function must be called once.

Reply:
Technically yes, however the bootstrap lifecycle is called by single-spa once prior to mount. My fear with rewording is making it sound like the implementor needs to invoke bootstrap.

Reply:
ahhh good point. Then Yeah I would leave it as is.